README.md

```diff
@@ -34,13 +32,14 @@ Sets the selected option to that which matches the provided value.  Updates the
 - `name`: the `<select>`'s `name` attribute's value. Used when reporting to parent form
 - `parent`: parent form reference
 - `options`: array/collection of options to render into the select box
-- `[groupOptions]`: use instead of `options` to generate &lt;optgroup&gt; elements within your &lt;select&gt;. If this is set, any values passed in `options` will be ignored and replaced with values coming from `groupOptions`.
+- `[groupOptions]`: use instead of `options` to generate <optgroup> elements within your <select>. If this is set, any values passed in `options` will be ignored and replaced with values coming from `groupOptions`.
```

Comment:
my .md preview seems to think this is a naughty change.

<img width="1054" alt="screen shot 2016-02-17 at 10 27 22 am" src="https://cloud.githubusercontent.com/assets/1003261/13118328/fb3ea990-d560-11e5-99c9-a3ccc4457797.png">